src/routes/search.ts

```diff
@@ -0,0 +1,28 @@
+import { FastifyInstance } from 'fastify';
+import { postFeedback } from '../integrations';
+import { isNullOrUndefined } from '../common/object';
+
+export interface FeedbackArgs {
+  value: number;
+  chunkId: string;
+}
+
+export default async function (fastify: FastifyInstance): Promise<void> {
+  fastify.post<{ Body: FeedbackArgs }>('/feedback', async (req, res) => {
```

Comment:
This should be a graphql mutation isn't it? @rebelchris

Reply:
Not sure actually, since these are only proxying request I suggested we keep it the same as we do for heimdall.
What would be pro/con of using mutation over this?

Reply:
It's needed for multiple reasons:
* Keep consistent with the rest of our codebase which is graphql
* Provide type-safety and clear expectations on input/output
* I want us to look at it as something other than a proxy per se. But more like a remote database. Imagine Magni is our database.

We somehow missed this point in the web team's DR, I guess. Good to note for next time

Sidenote: we don't proxy heimdall. We just call it as part of our boot endpoint.

Reply:
Alright would agree with that, just something that is "vague" and not standardized at the moment.
Maybe some day we can spend some time to setup some guidelines.

As in 
- prefer graphQL
- prefer graphORM
- etc

For heimdall we do have exclusions, like the route to check email etc, they are not via graphORM from top of my head.

Reply:
I agree that we need guidelines. For now anything API project related should use graphql and graphorm as much as possible to keep our conventions. Heimdall is available on sso.daily.dev so it's not proxy by api

Reply:
Amazing. Thank you both for the inputs. I shall make the changes.